Probesys/bileto
# new: Filter owned and unassigned tickets

Related to https://github.com/Probesys/bileto/issues/31 and https://github.com/Probesys/bileto/issues/41

Changes proposed in this pull request:

- Add a `uid` field to the User entity
- Refactor the listing of the tickets
- Add a navigation column to filter owned and unassigned tickets
- Adapt the placeholder when there are no tickets

How to test the feature manually:

1. Create some tickets (assigned to yourself, to another user or to no one)
2. Select the different filters on the left of the tickets page and check there are filtered correctly

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
